### DISCUSSION TITLE
【求助】我太小看对空六课的工作了

### DISCUSSION THREAD

**Discussion by Jinmaysong** [share121/inter-knot#2909] [其他]
以为刀耕行动很简单随便就可以赚一堆丁尼的我接受了防卫军的要求去控制尼尼微造成的乙太活性异常，后面才发现自己太小看枯败花园的危险性了，但现在钱也收了不干不行，有没有相关专业人员有什么办法安全脱身，不然我就只能连夜逃去外环了。
<img width="485" alt="niniway" src="https://github.com/user-attachments/assets/4a0b2b13-25e0-4dd2-b3bb-2e604cdcfe77">

**Comment by GrunderIndustries** [comment]
您可以购买我司的MQ-99产品 然后挂载MPBM对尼尼薇进行轰炸
另外如果客户真这么做了的话请提供一份威力报告 我们会报销您的无人机和MPBM钱

**Comment by HSOS6-LGDR** [comment]
业内人员表示：我不到啊<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" >打架不都是课长上的嘛<img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" >
